package.json

```diff
@@ -19,7 +19,9 @@
     "stylus": "~0.54.5",
     "uglify-js": "^2.7.5"
   },
-  "devDependencies": {},
+  "devDependencies": {
+    "jsonlint": "^1.6.2"
```

Comment:
> Again: I consider that backwards compatible. Whether or not your code passes the linter is not part of the public API. Even micro bumps can break that (when they fix broken rules), so...

Then you disagree with the vast majority of the npm community, who think that if it breaks _anything_, including your test suite, then it's a semver-major change. Adding new rules may not affect the API _interface_ but it certainly impacts API _behavior_. That makes it backwards-incompatible.